impeller/typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -104,13 +113,137 @@ static size_t OptimumAtlasSizeForFontGlyphPairs(
   return 0u;
 }
 
+/// Compute signed-distance field for an 8-bpp grayscale image (values greater
+/// than 127 are considered "on") For details of this algorithm, see "The 'dead
+/// reckoning' signed distance transform" [Grevera 2004]
+static void ConvertBitmapToSignedDistanceField(uint8_t* pixels,
```

Comment:
Its going to take me a bit to walk through this (and we should totally not block landing this because of it). But, can we fixup the style to match the style guide?

Reply:
Do you mean the comment bit? I tried to fix up the variable names.

Perhaps you mean the in-out pointer parameter? Doing this in-place avoids an extra allocation the size of the texture though. Maybe there's a more style-guide friendly way of doing that?